parts/linux/cloud-init/artifacts/components.json

```diff
@@ -48,8 +48,8 @@
       "multiArchVersionsV2": [
         {
           "renovateTag": "registry=https://mcr.microsoft.com, name=containernetworking/azure-cni",
-          "latestVersion": "v1.5.35",
-          "previousLatestVersion": "v1.5.32",
+          "latestVersion": "v1.4.56",
+          "previousLatestVersion": "v1.4.52",
```

Comment:
are we sure this image exists?

==> azure-arm: ctr: failed to resolve reference "mcr.microsoft.com/containernetworking/azure-cni:v1.4.52": mcr.microsoft.com/containernetworking/azure-cni:v1.4.52: not found

Reply:
I think those are versions for `azure-cns` not `azure-cni`? might need to swap to different versions for cni

Reply:
v1.4.52 and v1.4.54 does not exist in mcr for azure-cni.  Let me remove the azure-cni previous latest version from here and the prefetch.